src/ng/services/table.spec.js

```diff
@@ -0,0 +1,92 @@
+import Table from './table';
```

Comment:
By the way, should we have functionality like in this example? To access cell by index inside row and column classes?